Src/FluentAssertions/Common/InternalObjectExtensions.cs

```diff
@@ -3,7 +3,8 @@
 
 namespace FluentAssertions.Common
 {
-    public static class ObjectExtensions
+    // NOTE: These extension methods are covered by tests separately and therefore need to be public.
```

Comment:
We could add [`[InternalsVisibleTo]`](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=netcore-3.1)?

Reply:
We could, but then the Specs project must get a strong name, which requirement then bubbles to Chill, etc.

Reply:
😢